test/built-ins/Atomics/wait/symbol-for-timeout-throws.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2018 Amal Hussein. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-atomics.wait
+description: >
+  False timeout arg should result in a timeout value of 1
+info: |
+  Atomics.wait( typedArray, index, value, timeout )
+
+  4.Let q be ? ToNumber(timeout).
+    ...
+    Symbol	Throw a TypeError exception.
+features: [ Atomics ]
+---*/
+
+var sab = new SharedArrayBuffer(1024);
+var int32Array = new Int32Array(sab);
+
+assert.throws(TypeError, function() {
+  Atomics.wait(int32Array, 0, 0, Symbol('foo'))
+}, 'Symbol');
```

Comment:
all the cases throwing a TypeError after checking the timeout arg need to be protected in a agent broadcast as we won't be able to suspend the main host so it will throw a TypeError anyway. This protection will prevent false positives.

```
Let B be AgentCanSuspend().
If B is false, throw a TypeError exception.
```

Reply:
same for test/built-ins/Atomics/wait/posioned-object-for-timeout-throws.js